test/core/sitemap_request_list.test.ts

```diff
@@ -18,6 +18,41 @@ beforeAll(async () => {
 
     server = await startExpressAppPromise(app, 0);
     url = `http://localhost:${(server.address() as AddressInfo).port}`;
+    let throwError = true;
+
+    app.get('/sitemap-unreliable.xml', async (req, res) => {
+        if (throwError) {
+            throwError = !throwError;
```

Comment:
Could you replace this with `attemptCount` or something and check for `attemptCount % 2 === 0`? It took me a while to understand how the test interacts with that boolean.